### ISSUE TITLE
Void Config Option Does Not Seem to Work

### ISSUE THREAD

**Issue by Darkosto** [McJty/RFToolsDimensions#40]
Hello, 

I'm working on configuring RFTools for SkyFactory 3 and can't seem to figure out how to make the void world option work in newly created dimensions. I've made sure this config entry is changed:

```
# Set this to true if you want to make sure RFTools can only create void dimensions
B:voidOnly=true
```

Is there another option I can try to make this happen so only void type worlds appear?

MC Version: 1.10.2
RFTools Version: 1.10 - 5.50
RFtools Dimention: 1.10 4.40
Forge: 2095

Thank you!!
Darkosto

**Comment by McJty** [comment]
I will check it out. Perhaps I forgot to implement it in 1.10.2

**Comment by McJty** [comment]
Fixed in 4.41
